[Do not merge] iOS Swift Banner (#4368)
braze-inc/braze-docs: _includes/archive/swift_upgrade_banner.md

<br>
{% alert note %}

> These docs reference our Obj-C SDK which is now in maintenance mode. Please upgrade to our new [Swift SDK](https://github.com/braze-inc/braze-swift-sdk).

Reviewer: ```suggestion
These docs reference the deprecated Objective-C SDK (`appboy-ios-sdk`). Please upgrade to our new [Swift SDK](https://github.com/braze-inc/braze-swift-sdk).
```
